openeo/local/processing.py

```diff
@@ -34,7 +48,13 @@ def load_local_collection(*args, **kwargs):
     elif '.nc' in collection.suffixes:
         data = xr.open_dataset(kwargs['id'],chunks={},decode_coords='all').to_array(dim='bands') # Add decode_coords='all' if the crs as a band gives some issues
     elif '.tiff' in collection.suffixes or '.tif' in collection.suffixes:
-        data = rioxarray.open_rasterio(kwargs['id']).rename({'band':'bands'})
+        data = rioxarray.open_rasterio(kwargs['id'],chunks={},band_as_variable=True)
+        for d in data.data_vars:
+            descriptions = [v for k, v in data[d].attrs.items() if k.lower() == "description"]
+            if descriptions:
+                data = data.rename({d: descriptions[0]})
+        data = data.to_array(dim='bands')
     return data
 
-PROCESS_REGISTRY["load_collection"] = load_local_collection
\ No newline at end of file
+from openeo_processes_dask.specs import load_collection as load_collection_spec
+PROCESS_REGISTRY["load_collection"] = Process(spec=load_collection_spec, implementation=load_local_collection)
```

Comment:
I don't think it is necessary to do the `from openeo_processes_dask.specs import load_collection as load_collection_spec` import, you can just define load_collection with

```python
PROCESS_REGISTRY["load_collection"] = Process(
    spec=openeo_processes_dask.specs.load_collection, 
    implementation=load_local_collection,
)
```